spi/split-brain/src/main/java/ComposedHitsAndCreationTimeMapMergePolicyExample.java

```diff
@@ -19,7 +19,7 @@
 import com.hazelcast.config.MapConfig;
 import com.hazelcast.config.MergePolicyConfig;
 import com.hazelcast.core.Hazelcast;
-import com.hazelcast.instance.HazelcastInstanceFactory;
+import com.hazelcast.instance.impl.HazelcastInstanceFactory;
```

Comment:
Again, reference to `internal`/`impl` API

Reply:
Can you suggest alternative, I could not catch `internal` in package naming for `HazelcastInstanceFactory `

Reply:
cc @Holmistr

Reply:
Hi @lazerion, the correct API to use is `Hazelcast.newHazelcastInstance()`.